src/pqm/__init__.py

```diff
@@ -1,10 +1,12 @@
 from .pqm import pqm_pvalue
+from .pqm import pqm_chi2
```

Comment:
they can both go on one line
```suggestion
from .pqm import pqm_pvalue, pqm_chi2
```